### ISSUE TITLE
MOB: Snow Leopard

### ISSUE THREAD

**Issue by zwei7** [HellscreamWoW/Tracker#3674]
This MOB jumps 10m in the air toward you when you attack it from afar.
![wowscrnshot_111915_184835](https://cloud.githubusercontent.com/assets/15259298/11288050/57f0bb10-8eee-11e5-9c1e-878f179eefcd.jpg)

**Comment by pourisse** [comment]
Yep, and when you kill him when his up, you can't reach the loot.
